bridges/snowbridge/pallets/inbound-queue-v2/src/lib.rs

```diff
@@ -230,7 +181,7 @@ pub mod pallet {
 		#[pallet::call_index(0)]
 		#[pallet::weight(T::WeightInfo::submit())]
 		pub fn submit(origin: OriginFor<T>, message: Message) -> DispatchResult {
-			let who = ensure_signed(origin)?;
+			let _who = ensure_signed(origin)?;
```

Comment:
Is this simplification not possible?
```suggestion
			ensure_signed(origin)?;
```

Reply:
I kept the variable because it will be used for the rewards ledger once the unified rewards PR is merged: https://github.com/claravanstaden/polkadot-sdk/pull/3/files#diff-bfe761b371a553619edf5be2dd792c45bd14f1a916e51b77f5b77b9395d92cdaR209